exe/shaman

```diff
@@ -27,13 +27,13 @@ command :deploy do |c|
   c.description = 'Deploy a release to specified environment'
   c.option '-m', '--message MESSAGE', String, 'Changelog message'
   c.option '-f', '--file FILE', String, 'Release path'
-  c.option '-p', '--proguard FILE', String, 'Add aditional proguard mapping'
```

Comment:
What's our release plan here? This is a breaking change so a major version upgrade?

Reply:
Initially yes that was my idea, but now you got me thinking if it would be better to release a minor version first without this change. Then release a major version with this change, planned refactorings, and potential feedback with this new release.